feature/select-subject-age-group/src/main/java/com/simprints/feature/selectagegroup/screen/BuildAgeGroupsUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package com.simprints.feature.selectagegroup.screen
+
+import com.simprints.infra.config.store.ConfigRepository
+import com.simprints.infra.config.store.models.AgeGroup
+import com.simprints.infra.config.store.models.sortedUniqueAgeGroups
+import javax.inject.Inject
+
+internal class BuildAgeGroupsUseCase @Inject constructor(
+    private val configurationRepo: ConfigRepository,
+) {
+
+    /**
```

Comment:
Is this code comment still valid?

Reply:
It is still valid, I just removed the part that was referring to display text.